kernel/build/config-amd64

```diff
@@ -5474,7 +5474,7 @@ CONFIG_LEDS_TRIGGERS=y
 # Simple LED drivers
 #
 # CONFIG_ACCESSIBILITY is not set
-CONFIG_INFINIBAND=y
+CONFIG_INFINIBAND=m
```

Comment:
I don't think we can accept such changes close to the release of 1.10. Can you try to minimize changes so that only the stuff you need is enabled?

Reply:
This will be in a seperated PR.